src/test/java/org/gridsuite/actions/server/ContingencyListControllerTest.java

```diff
@@ -865,7 +920,8 @@ public void copyFormToScriptTest() throws Exception {
         UUID firstUUID = addNewFormContingencyList(form);
 
         // new script from form
-        String res = mvc.perform(post("/" + VERSION + "/form-contingency-lists/" + firstUUID + "/new-script"))
+        String res = mvc.perform(post("/" + VERSION + "/form-contingency-lists/" + firstUUID + "/new-script")
+                .header(USER_ID_HEADER, "userId"))
```

Comment:
Why is userId needed here ? newScriptFromFormContingencyList doesn't request it in the controller

Reply:
removed